### PR TITLE
Add Demos Page to the Website

### PR DESCRIPTION
**Pull request by ufukyilmaz**
Add a new page including the demo applications to the website.

### CODE REVIEW

File: site/website/package.json [hunk 1]
```diff
@@ -31,6 +31,8 @@
     "wait-on": "^4.0.0"
   },
   "dependencies": {
-    "concurrently": "^5.1.0"
+    "concurrently": "^5.1.0",
+    "@material-ui/core": "^4.9.10",
```
Comment (eminn) [inline site/website/package.json:35]: Do we really need Material UI just for Card component ?

Comment (ufukyilmaz) [reply site/website/package.json]: I removed the unnecessary dependencies.

File: site/website/pages/en/demos.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
+const React = require('react');
+const CompLibrary = require('../../core/CompLibrary');
+const Container = CompLibrary.Container;
+const MarkdownBlock = CompLibrary.MarkdownBlock;
+
+class Demos extends React.Component {
+
+    render() {
+        const {config: siteConfig, language = ''} = this.props;
+        const cardStyle = {
```
Comment (cangencer) [inline site/website/pages/en/demos.js:10]: would it better to move these to the css?

Comment (ufukyilmaz) [reply site/website/pages/en/demos.js]: I moved these styles to a jss file.

File: site/website/pages/en/demos.js [hunk 3]
```diff
@@ -2,60 +2,20 @@ const React = require('react');
 const CompLibrary = require('../../core/CompLibrary');
 const Container = CompLibrary.Container;
 const MarkdownBlock = CompLibrary.MarkdownBlock;
+const cardStyle = require(process.cwd() + "/static/jss/card.js");
```
Comment (cangencer) [inline site/website/pages/en/demos.js:5]: shouldn't you use relative path here?

Comment (ufukyilmaz) [reply site/website/pages/en/demos.js:5]: When I tried to use a relative path, I got an error for every path I tried. I think the docusaurus does not support it. When I looked at how to import this jss component, I came across [this](https://docusaurus.io/docs/en/api-pages#page-require-paths) and applied it.

**Comment by cangencer** [comment]
@eminn could you review?

**Pull request merged**